command/agent.go

```diff
@@ -410,6 +410,15 @@ func (cmd *AgentCommand) readConfig() *agent.Config {
 		cmd.UI.Error(fmt.Sprintf("WARNING: Expect Mode enabled, expecting %d servers", cfg.BootstrapExpect))
 	}
 
+	// Warn if we are expecting an even number of servers
+	if cfg.BootstrapExpect != 0 && cfg.BootstrapExpect%2 == 0 {
+		if cfg.BootstrapExpect == 2 {
+			cmd.UI.Error("WARNING: A cluster with 2 servers will provide no failure tolerance")
```

Comment:
Can add the link to See https://www.consul.io/docs/internals/consensus.html#deployment-table here similar to the other warning message.